metatron-app/metatron-discovery
# F#3342 treat string measure

### Description
Druid's virtual expression can treat String result. For example, you can get the result of time_format(max(__time), 'yyyy-MM-dd') without error. But Metatron Discovery can only treat double measure. So we need to treat String type measure.

**Related Issue** : 
https://github.com/metatron-app/metatron-discovery/issues/3342


### How Has This Been Tested?
1. Create Caculation measure column in Druid.
The fomula looks like this : TIME_FORMAT( maxof(__time), 'yyyy-MM-dd' )
2. Draw Gridchart. Place caculation column on value shelf.
3. Make sure the date is represented correctly.

#### Need additional checks?


### Types of changes

- [ x ] Breaking change (fix or feature that would cause existing functionality to change)


### Checklist:

- [ x ] I have read the **CONTRIBUTING** document.
- [ x ] My code follows the code style of this project.
- [ x ] I have added tests to cover my changes.

### Additional Context
